src/plots/cartesian/axes.js

```diff
@@ -1799,12 +1811,18 @@ function numFormat(v, ax, fmtoverride, hover) {
         // 0 is just 0, but may get exponent if it's the last tick
         v = '0';
         isNeg = false;
-    }
-    else {
+    } else {
```

Comment:
We should add an `else` example in there. Don't think there's one there yet!

My main argument for the `} else {` is so that you don't accidentally put code between two clauses where you might have something like:

``` javascript
if(meaningOfLife === 42) dontPanic();
// this is a reference to an entertaining book

// for some reason there's space between the clauses
else panic();
```

I've run into this in a [few](https://github.com/plotly/plotly.js/blob/master/src/plots/cartesian/axes.js#L1475) [places](https://github.com/plotly/plotly.js/blob/master/src/plots/cartesian/axes.js#L838) and accidentally put code between them, then promptly :confused:'d when browserify breaks because of the syntax issue. It's definitely my fault, but it's a lot harder to make the mistake when the clauses are tied together on the line between them.